src/main/java/com/adobe/pdfservices/operation/samples/exportpdf/ExportPDFToDOCXWithOCROption.java

```diff
@@ -8,59 +8,62 @@
  * then your use, modification, or distribution of it requires the prior
  * written permission of Adobe.
  */
-
 package com.adobe.pdfservices.operation.samples.exportpdf;
 
-import java.io.IOException;
-
 import com.adobe.pdfservices.operation.ExecutionContext;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
 import com.adobe.pdfservices.operation.auth.Credentials;
 import com.adobe.pdfservices.operation.exception.SdkException;
 import com.adobe.pdfservices.operation.exception.ServiceApiException;
 import com.adobe.pdfservices.operation.exception.ServiceUsageException;
 import com.adobe.pdfservices.operation.io.FileRef;
 import com.adobe.pdfservices.operation.pdfops.ExportPDFOperation;
+import com.adobe.pdfservices.operation.pdfops.options.exportpdf.ExportOCRLocale;
+import com.adobe.pdfservices.operation.pdfops.options.exportpdf.ExportPDFOptions;
 import com.adobe.pdfservices.operation.pdfops.options.exportpdf.ExportPDFTargetFormat;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
 
 /**
- * This sample illustrates how to export a PDF file to JPEG.
- * <p>
- * Note that exporting a PDF file to an image format results in a ZIP archive containing one image per page.
+ * This sample illustrates how to export a PDF file to a Word (DOCX) file. The OCR processing is also performed on the input PDF file to extract text from images in the document.
  * <p>
  * Refer to README.md for instructions on how to run the samples.
  */
-public class ExportPDFToJPEG {
+public class ExportPDFToDOCXWithOCROption {
 
     // Initialize the logger.
-    private static final Logger LOGGER = LoggerFactory.getLogger(ExportPDFToJPEG.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(ExportPDFToDOCXWithOCROption.class);
 
     public static void main(String[] args) {
+
         try {
 
             // Initial setup, create credentials instance.
             Credentials credentials = Credentials.serviceAccountCredentialsBuilder()
                     .fromFile("pdfservices-api-credentials.json")
                     .build();
-
             //Create an ExecutionContext using credentials and create a new operation instance.
             ExecutionContext executionContext = ExecutionContext.create(credentials);
-            ExportPDFOperation exportPdfOperation = ExportPDFOperation.createNew(ExportPDFTargetFormat.JPEG);
+            ExportPDFOperation exportPdfOperation = ExportPDFOperation.createNew(ExportPDFTargetFormat.DOCX);
 
             // Set operation input from a source file.
-            FileRef sourceFileRef = FileRef.createFromLocalFile("src/main/resources/exportPDFToImageInput.pdf");
+            FileRef sourceFileRef = FileRef.createFromLocalFile("src/main/resources/exportPDFInput.pdf");
             exportPdfOperation.setInput(sourceFileRef);
 
+            //Set export pdf ocr option
```

Comment:
Create a new ExportPDFOptions instance from the specified OCR locale and set it into the operation